README.md

```diff
@@ -481,6 +485,69 @@ The verified in-toto statement may be written to stdout with the
 
 Note that `--source-uri` supports GitHub repository URIs like `github.com/$OWNER/$REPO` when the build was enabled with a Cloud Build [GitHub trigger](https://cloud.google.com/build/docs/automating-builds/github/build-repos-from-github). Otherwise, the build provenance will contain the name of the Cloud Storage bucket used to host the source files, usually of the form `gs://[PROJECT_ID]_cloudbuild/source` (see [Running build](https://cloud.google.com/build/docs/running-builds/submit-build-via-cli-api#running_builds)). We recommend using GitHub triggers in order to preserve the source provenance and valiate that the source came from an expected, version-controlled repository. You _may_ match on the fully-qualified tar like `gs://[PROJECT_ID]_cloudbuild/source/1665165360.279777-955d1904741e4bbeb3461080299e929a.tgz`.
 
+## Verification Summary Attestations (VSA)
+
+We have support for [verifying](https://slsa.dev/spec/v1.1/verification_summary#how-to-verify) VSAs.
+Rather than passing in filepaths as arguments, we allow passing in mulitple `--subject-digest` cli options, to
+accomodate subjects that are not simple-files.
+
+
+The verify-vsa command
+
+```shell
+$ slsa-verifier verify-vsa --help
+Verifies SLSA VSAs for the given subject-digests
+
+Usage:
+  slsa-verifier verify-vsa [flags] subject-digest [subject-digest...]
+
+Flags:
+      --attestations-path string      path to a file containing the attestations
+  -h, --help                          help for verify-vsa
+      --print-attestation             [optional] print the contents of attestation to stdout
+      --public-key-signing-hash-algo string   [optional] the hash algorithm used to compute the digest to be signed, one of SHA256 [default], SHA384, or SHA512
+      --public-key-id string          [optional] the ID of the public key, defaults to the SHA256 digest of the base64-encoded public key
+      --public-key-path string        path to a public key file
+      --resource-uri string           the resource URI to be verified
+      --subject-digest stringArray    the digests to be verified. Pass multiple digests by repeating the flag. e.g. <digest type>:<digest value>
+      --verified-levels strings       [optional] the levels of verification to be performed, comma-separated. e.g., 'SLSA_BUILD_LEVEL_2,FEDRAMP_LOW'
```

Comment:
It's not a SLSA level, but the specs leave open the possibility of additional "tracks" that can be included in verifiedLvels.

Reply:
those additional tracks do _not_ exist today. I don't see value in committing to something that's not landed yet. If the specs said "arbitrary values can be used, not just verified levels", then I'd make sense. In particular FEDRAMP is _not_ a valid track level, so should ,not be accepted since it does not conform to the current specs.
What the use case for this atm?

Reply:
> Users MAY use custom values here but MUST NOT use custom values starting with SLSA_.

It's purely an example. I know FEDRAMP isn't a public track. But we still have to allow custom, non-public tracks. I just chose FEDRAMP because it seems like a [plausible](https://www.fedramp.gov/baselines/) track that an org might have in private.

I could change this example to say `MY_CUSTOM_TRACK_3`. wdyt?

Reply:
I'd just drop FEDRAMP and stick with the defined SlsaResults, and maybe add a later comment that it can be a custom result as well.

Reply:
> Users MAY use custom values here but MUST NOT use custom values starting with SLSA_.

Good point. Somehow I always end up on v0.1 of the specs because they are the default page shown and this line was not there :/

sidenote: the levels have some ordering we need to understand. If the user passes BUILD_LEVEL2 and the VSA says it's BUILD_LEVEL3, then it should succeed, because 3 >= 2. With unknown arbitrary names, we won't be able do that (nothing we can do about it)

Reply:
Talked offline, its okay to keep the FEDRAMP exampe.